lib/opendax/vault.rb

```diff
@@ -56,5 +58,24 @@ def setup
       secrets('enable', ['kv'], '-path=secret -version=1')
       vault_root_token
     end
+
+    def load_policies(deployment_name, vault_root_token)
+      puts '----- Vault login -----'
+      vault_exec("vault login #{vault_root_token}")
+
+      tokens = {}
+      POLICIES_NAMES.each do |policy|
+        name = "#{deployment_name.downcase}_#{policy}"
+
+        puts "Loading #{name} policy..."
+        vault_exec("vault policy write #{name} /tmp/policies/#{policy}.hcl")
+
+        puts "Creating the #{name} token..."
+        vault_token_create_output = YAML.safe_load(vault_exec("vault token create -policy=#{name} -renewable=true -ttl=864000 -format=yaml"))
```

Comment:
you need to specify the duration here also to make sure the token will be able to renew indefinitely.
try the following args: -ttl=240h -duration=240h

Reply:
Do you mean "period" flag?